bridgefy/bridgefy_flutter
# Bridgefy Flutter 1.0.1 (#10)

* chore: start from template

* chore: add ios framework manually

* chore: first method tests

* chore: allow flutter debugging

* feat: start data type conversions

* feat: exception throws

* fix: fill in all delegate methods

* chore: complete interface

* feat: complete delegate methods

* chore: add next methods

* fix: tweak object initializers

* chore: upgrade bridgefy framework

* fix: uuid and data interfaces

* chore: add android gradle dep

* chore: add android structure

* refactor: use flutter errors

* fix: unify error handling

* feat: kotlin delegates and errors

* chore: gradle files

* fix: mark android exclusive events

* fix: gradle settings

* fix: use anonymous object for delegate

* chore: launch configs

* chore: main dart

* chore: api key template

* test: platform methods

* test: all delegate methods

* docs: README first version

* chore: clean up some more

* feat: complementar test app (#2)

* chore: upgrade dependency on work-runtime-ktx

* feat: add required libraries

* feat: add function to check permissions

* chore: use bridgefy red color

* fix: android 13 permissions

* fix: SingleChildScrollView overflowed using expanded widget

* chore: Upgrade Android library 0.4.0-SNAPSHOT -> 1.0.0

* doc: Update README.md

* Update pubspec.yaml

* chore: repackaging bridgefy plugin

* chore: Fixed one type argument expected in init method

* chore: added necessary else branch in Transmission Mode

* chore: variable code was initialized

* fix: android config

* doc: add some instructions

* Update build.gradle

* fix: upgrade dep to 1.0.1, update README

* chore: bump minor version in pubspec

* Release/1.0.1 (#11)

* chore: Change log updated

* chore: Release version 1.0.1

* chore: metadata added in the package's pubspec

* chore: rename LICENSE.md -> LICENSE

* chore: 5 topics listed.

* chore: rename topic mesh network -> mesh

* chore: rename topic BLE -> ble Bluetooth -> bluetooth
